pkg/client/group_mgmt_client.go

```diff
@@ -84,6 +88,23 @@ func (client *GroupMgmtClient) EnableDebug() {
 	client.Client.SetDebug(true)
 }
 
+// refreshSessionToken : refresh session token
+func (client *GroupMgmtClient) refreshSessionToken() (bool, error) {
+	// Invalid credential has empty token
+	if client.SessionToken != "" {
```

Comment:
I've lost track of what this condition is for.  I'm concerned that if the login fails at line 96, we cannot recover.  Is there any update on using RetryConditions?